lib/hexpm/accounts/email.ex

```diff
@@ -65,6 +81,14 @@ defmodule Hexpm.Accounts.Email do
     |> unique_constraint(:email, name: "emails_email_key", message: "already in use")
   end
 
+  def update_email(email, new_address) do
+    change(email, %{email: new_address})
+    |> validate_required(~w(email)a)
```

Comment:
Extract a private function for these four email-related validations/constraints?

Reply:
Yes, they are duplicated enough.

Reply:
Done in 7b736d8af5e71af7762b0bc2e0505ab34f48438e